test/unit/npm-dependncies.spec.js

```diff
@@ -0,0 +1,58 @@
+const npmDependencies = require("../../src/dynamicTypes/npmDependencies");
+
+describe("npm dependencies", () => {
+  describe("should load types for npm modules", () => {
+    it("get default type when request raw", () => {
+      const rawNpmType = npmDependencies.getRaw("react");
+      expect(rawNpmType).toMatchSnapshot();
+    });
+
+    it("get default type when request files and the namespaces is not @wix", async () => {
+      const npmDtsFiles = await npmDependencies.getFiles({
+        react: "16.8.6",
+        "react-dom": "16.8.6"
+      });
+      expect(npmDtsFiles).toMatchSnapshot();
```

Comment:
is it common in wix to use jest snapshot matching? I never used it..

Reply:
I'll refactor that 🙏